packages/atst/src/cli.ts

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
```

Comment:
tsup is smart enough to make the js output of this file executable from this being in file